### ISSUE TITLE
KeyError while processing url_entity

### ISSUE THREAD

**Issue by herrwusel** [robertoszek/pleroma-bot#48]
Here is the log:
```
$ pleroma-bot 

                        `^y6gB@@BBQA{,
                      :fB@@@@@@BBBBBQgU"
                    `f@@@@@@@@BBBBQgg80H~
                    H@@B@BB@BBBB#Qgg&0RNT
                   z@@&B@BBBBBBQgg80RD6HK
                  ;@@@QB@BBBB#Qgg&0RN6WqS
                  q@@@@@BBBBQgg80RN6HAqSo          _             _
                 z@@@@BBBB#Qg8&0RN6WqSUhr         | |           | |
               -H@@@@BBBBQQg80RD6HAqSKh(       ___| |_ ___  _ __| | __
              rB@@@BBBB#6Lm00DN6WqSUhfv       / __| __/ _ \| '__| |/ /
             f@@@@BBBBf= |0RD6HAqSKhfv        \__ \ || (_) | |  |   <
           =g@@@BBBBF=  "RDN6WqSUhff{         |___/\__\___/|_|  |_|\_|
          c@@@@BBgu_   ~WD9HAqSKhfkl`
        _6@@@BBNr     'qN6WqSUhhfXI'     .                           .       .
       rB@@@B0r      `S6HAqSKhfkoCr  ,-. |  ,-. ,-. ,-. ,-,-. ,-.    |-. ,-. |-
     `X@@@BQx       `I6WASShhfXFIy_  | | |  |-' |   | | | | | ,-| -- | | | | |
    _g@@@Q\`        JHAqSKhfXoCwJz_  |-' `' `-' '   `-' ' ' ' `-^    `-' `-' `'
   rB@@#x`         }WASShhfXsIyzuu,  |
 `y@@&|          .IAqSKhfXoCwJzu1lr  '
`D@&|           :KqSUhffXsIyzuu1llc,
ff=            `==:::""",,,,________


ℹ 2021-06-04 15:00:48,547 - pleroma_bot - INFO - ====================================== 
ℹ 2021-06-04 15:00:48,547 - pleroma_bot - INFO - Processing user:       xxx 
ℹ 2021-06-04 15:00:50,202 - pleroma_bot - INFO - tweet count:    16 
Processing tweets...  |✖ 2021-06-04 15:00:50,508 - pleroma_bot - ERROR - Exception occurred (cli.py:482) 
Traceback (most recent call last):
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/_processing.py", line 228, in _expand_urls
    for url_entity in tweet["entities"]["urls"]:
KeyError: 'entities'

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/cli.py", line 454, in main
    tweets_to_post = user.process_tweets(tweets)
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/_utils.py", line 93, in wrapper
    input_thread.join()
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/_utils.py", line 50, in join
    raise self.exc
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/_utils.py", line 38, in run
    self.ret = self._target(*self._args, **self._kwargs)
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/_utils.py", line 84, in <lambda>
    target=lambda q, *arg1, **kwarg1: q.put(fct(*arg1, **kwarg1)),
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/_processing.py", line 70, in process_tweets
    tweet["text"] = _expand_urls(self, tweet)
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/pleroma_bot/_processing.py", line 250, in _expand_urls
    response = session.head(match.group(), allow_redirects=True)
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/requests/sessions.py", line 577, in head
    return self.request('HEAD', url, **kwargs)
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/requests/sessions.py", line 528, in request
    prep = self.prepare_request(req)
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/requests/sessions.py", line 466, in prepare_request
    hooks=merge_hooks(request.hooks, self.hooks),
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/requests/models.py", line 316, in prepare
    self.prepare_url(url, params)
  File "/home/mstdn/bot/venv/lib/python3.7/site-packages/requests/models.py", line 390, in prepare_url
    raise MissingSchema(error)
requests.exceptions.MissingSchema: Invalid URL 'Twitch.tv/xxx': No schema supplied. Perhaps you meant http://Twitch.tv/xxx?

```

**Comment by robertoszek** [comment]
Hi!
It seems I didn't take into account the protocol not being present (http://, https://) when falling back to regex to match URLs in the tweet's text.
It took me a while to reproduce, Twitter does a good job most of the time replacing plain text URLs with their shortened link and including the appropriate metadata for the url entities.

I have published a new version (0.8.9) on TestPyPI with the fix, you can install it this way:
```shell
pip install --index-url https://test.pypi.org/simple/ --extra-index-url https://pypi.org/simple pleroma-bot --upgrade
```
Let me know if it works for your case and I will include it on the next official release.

**Comment by herrwusel** [comment]
That was fast! And it's working - thanks!

I found the tweet that might have been the issue:
```
Htttp://Twitch.tv/...
```
